### PR TITLE
Update decompiler, fix kazas crash

### PR DESCRIPTION
**Pull request by avionanx**
Should replace TODOs with branch parameters in the future

### CODE REVIEW

File: src/main/java/legend/game/debugger/ScriptLiveDebuggerController.java [hunk 1]
```diff
@@ -85,8 +85,9 @@ private void decompile() {
         this.btnResume.setDisable(true);
         this.btnStep.setDisable(true);
       }
-
-      this.tokens = this.disassembler.disassemble(state.scriptPtr_14.data);
+      //TODO
```
Comment (LordMonoxide) [inline src/main/java/legend/game/debugger/ScriptLiveDebuggerController.java:88]: Code style:
- Keep blank line after closing }
- Add a note after TODO to explain why, create a ticket and include the ticket number in the TODO (e.g. `Add support for forced decomp branches #12345)
- All variables should be final unless absolutely necessary
- Should be a space after commas in parameter lists

Our IntelliJ inspections should warn about the non-final var. Alt+enter to apply quick fixes. You can reformat code using our code style by highlighting it and pressing ctrl+alt+L.

File: src/main/java/legend/game/unpacker/scripts/ScriptPatcher.java [hunk 2]
```diff
@@ -135,7 +135,9 @@ public void unpatchFile(final Path sourceFile, final Path patchFile) throws IOEx
   }
 
   private List<String> decompile(final byte[] data) {
-    final Script script = this.disassembler.disassemble(data);
+    //TODO
```
Comment (LordMonoxide) [inline src/main/java/legend/game/unpacker/scripts/ScriptPatcher.java:138]: See previous comment, most of it applies here too

**Review by LordMonoxide — CHANGES_REQUESTED** [review]

File: src/main/java/legend/game/debugger/ScriptLiveDebuggerController.java (hunk 1 above)
Comment (avionanx) [reply src/main/java/legend/game/debugger/ScriptLiveDebuggerController.java]: Should be done

File: src/main/java/legend/game/unpacker/scripts/ScriptPatcher.java (hunk 2 above)
Comment (avionanx) [reply src/main/java/legend/game/unpacker/scripts/ScriptPatcher.java]: Also should be done now

**Review by LordMonoxide — APPROVED** [review]

**Pull request merged**